app/views/registration/summary.scala.html

```diff
@@ -35,96 +35,118 @@ <h1 class="h1-heading">@Messages("summary.heading")</h1>
+                        } else {
+                            @Messages("org.details.corpbody.view")
+                        }
                     }
-                }
-            </span>
-        </div>
-        <!-- Org name -->
-        <div class="tabular-data__entry tabular-data__entry--grouped divider--bottom">
-            <span class="tabular-data__heading tabular-data__heading--label" id="title-companyName">@Messages("summary.companydetails.name")</span>
-            <span class="tabular-data__data-1 tabular-data__data-1--2-col" id="value-companyName">@data.organisationDetails.companyName</span>
-        </div>
-        <div class="tabular-data__entry tabular-data__entry--grouped divider--bottom">
-            <span class="tabular-data__heading tabular-data__heading--label" id=@{s"title-${utrName}Number"}>
-                @Messages(s"summary.companydetails.${utrName}")
-            </span>
-            <span class="tabular-data__data-1 tabular-data__data-1--2-col" id=@{s"value-${utrName}Number"}>@data.organisationDetails.ctrNumber</span>
-        </div>
-        <!-- Org ref numbers -->
-        <div class="tabular-data__entry tabular-data__entry--grouped divider--bottom">
-            <span class="tabular-data__heading tabular-data__heading--label" id="title-fsrRefNumber">@Messages("summary.companydetails.fsrref")</span>
-            <span class="tabular-data__data-1" id="value-fsrRefNumber">@data.tradingDetails.fsrRefNumber</span>
-            <a class="tabular-data__data-2" href="@routes.TradingDetailsController.get?returnUrl=@routes.SummaryController.get" id="change-organisationNumbers-fsrRefNumber" data-journey-click="link - click:Check your answers:Change your FCA reference number">
-                <span aria-hidden="true">@Messages("summary.change")</span>
-                <span class="visuallyhidden">@Messages("summary.companydetails.fsrref.change")</span>
-            </a>
-        </div>
-        <div class="tabular-data__entry tabular-data__entry--grouped divider--bottom">
-            <span class="tabular-data__heading tabular-data__heading--label" id="title-isaProviderRefNumber">@Messages("summary.companydetails.isaref")</span>
-            <span class="tabular-data__data-1" id="value-isaProviderRefNumber">@data.tradingDetails.isaProviderRefNumber</span>
-            <a class="tabular-data__data-2" href="@routes.TradingDetailsController.get?returnUrl=@routes.SummaryController.get" id="change-organisationNumbers-isaProviderRefNumber" data-journey-click="link - click:Check your answers:Change your ISA manager reference">
-                <span aria-hidden="true">@Messages("summary.change")</span>
-                <span class="visuallyhidden">@Messages("summary.companydetails.isaref.change")</span>
-            </a>
-        </div>
+                </dd>
+                <dd class="govuk-summary-list__actions"></dd>
+            </div>
+            <div class="govuk-summary-list__row">
```

Comment:
Probably worth pulling this structure out.

Reply:
Structure is inconsistent, may not be worth doing this. (Particularly the size of the link properties)